tests/test_modeling_reformer.py

```diff
@@ -829,7 +844,7 @@ def test_local_layer_forward_complex(self):
         reformer_output = layer(prev_attn_output=hidden_states, hidden_states=hidden_states, attention_mask=attn_mask,)
         output_slice = reformer_output.hidden_states[0, 0, :5]
         expected_output_slice = torch.tensor(
-            [1.5476, -1.9020, -0.9902, 1.5013, -0.1950], dtype=torch.float, device=torch_device,
+            [1.4750, -2.0235, -0.9743, 1.4463, -0.1269], dtype=torch.float, device=torch_device,
```

Comment:
This change changes the resulting output?

Reply:
Yeah, good that you mention that - I forgot to adapt the other tests. 

What is happening here is that there was an error previously in the tests:
The input hidden states only had a sequence length of 4 and the chunk length is also 4 but `local_num_chunks_before` was set to 1 which would have meant to attend to the previous chunk as well as its own chunk. This is not possible if there is only one chunk. So I changed `local_num_chunkes_before` to 0. This only affected this test because the attention mask in local self attention depends on the function `self._look_adjacent()`, where as the one in LSH self attention does not.

Reply:
Alright, that makes sense! Thanks for the explanation.